src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
```

Comment:
Please add an access modifier to your Ball class. It's a good practice to define the scope of your classes explicitly, even if you want to use the default package-private access level.

Reply:
> Please add an access modifier to your Ball class. It's a good practice to define the scope of your classes explicitly, even if you want to use the default package-private access level.

I have a access modifier public in this class.